litrpc/lit-accounts.yaml

```diff
@@ -15,3 +15,9 @@ http:
       get: "/v1/accounts"
     - selector: litrpc.Accounts.RemoveAccount
       delete: "/v1/accounts/{id}"
```

Comment:
this should be done in the previous commit to keep the RPC check happy for each commit